### PR TITLE
deathmatch jail map

### PR DESCRIPTION
**Pull request by mc-oofert**
## About The Pull Request

image

cooked

24 players max
2 loadouts:
prisoner - prisoner with shiv
guard - essentially a prisoner with a vest and helmet and a stun baton with no cell

## Why It's Good For The Game

prisoners killing eachother in cafeteria = fun

## Changelog
:cl:
add: deathmatch prison map
/:cl:

### CODE REVIEW

**Comment by Vect0r2** [comment]
Hello! I would just like to let you know I am looking at this at the moment! We have quite a bit more setup to go though before I will seriously look though this and make sure it's mergable (and hopefully get some better mappers to look over it). But I will let you know when we are ready to start looking though it.

File: _maps/deathmatch/perma_jail.dmm
Comment (grungussuss) [inline _maps/deathmatch/perma_jail.dmm:1]: image
image
This light looks really odd ingame.

**Review by grungussuss — CHANGES_REQUESTED** [review]
You need to modularize the code changes, if you can't modularize it then place a comment saying that it's a `wallstation edit`

File: _maps/deathmatch/perma_jail.dmm
Comment (mc-oofert) [reply _maps/deathmatch/perma_jail.dmm:1]: called an illusion

**Pull request closed without merging**